packages/psql/examples/bun.ts

```diff
@@ -0,0 +1,42 @@
+import { Bot, Context, session, SessionFlavor } from "grammy";
+import { PsqlAdapter } from '@grammyjs/storage-psql';
+import { SQL } from "bun"
+
+interface SessionData {
+  counter: number;
+}
+
+type MyContext = Context & SessionFlavor<SessionData>;
+
+async function bootstrap() {
+  const sql = new SQL();
+
+  await sql.connect();
```

Comment:
How does it know where to connect to? You'll need to add at least a few comments, but rather explain things in the markdown files

Reply:
Bun reads `DATABASE_URL` by default, it is linked in bun docs.

https://bun.com/docs/runtime/sql#database-environment-variables